test/integration/action/action-integration.spec.js

```diff
@@ -105,10 +106,12 @@ describe('Action Integration Tests', function() {
       isDev,
       logger,
       btcdSettingsDir: BTCD_SETTINGS_DIR,
+      miningAddress: BTCD_MINING_ADDRESS,
```

Comment:
Why do we need to set the miningAddress? We were mining block before without it in the integration tests.

Reply:
When? When I leave out that line, I get this error: 
![deepinscreenshot_select-area_20181019125256](https://user-images.githubusercontent.com/10647624/47240662-f0ea1900-d39d-11e8-8a9a-999ce6d22ac4.png)